alacritty/src/event.rs

```diff
@@ -35,10 +33,10 @@ use alacritty_terminal::term::{ClipboardType, SizeInfo, Term, TermMode};
 #[cfg(not(windows))]
 use alacritty_terminal::tty;
 
-use crate::cli::Options as CliOptions;
+use crate::cli::{Options as CliOptions, TerminalOptions as TerminalCLIOptions};
```

Comment:
```suggestion
use crate::cli::{Options as CliOptions, TerminalOptions as TerminalCliOptions};
```

Correct capitalization in Rust is camelcase even for acronyms.